### ISSUE TITLE
[BUG]Verify SSH Login Access With Admin User

### ISSUE THREAD

**Issue by justin0309** [openbmc/openbmc-test-automation#2232]
This commit modification will cause test fail. Without these two lines of code, the test will pass.
https://github.com/openbmc/openbmc-test-automation/commit/e8015b3e14fdfd0303af57b4aa9fa99837d14421

Redfish.Login
Redfish.Delete  /redfish/v1/AccountService/Accounts/new_admin

![image](https://github.com/openbmc/openbmc-test-automation/assets/109942943/a0058a8d-b733-4cae-ac77-6dbea626e932)

**Comment by gkeishin** [comment]
@rahulmah or @prkatti1  can you respond to this ?

**Comment by gkeishin** [comment]
The test intends to delete the newly created new_admin password after the SSH validation is done.  Can you check if the user has been created?

**Comment by justin0309** [comment]
> The test intends to delete the newly created new_admin password after the SSH validation is done. Can you check if the user has been created?

![image](https://github.com/openbmc/openbmc-test-automation/assets/109942943/7c8a20cc-b035-4018-96bf-d65284e35525)

**Comment by prkatti1** [comment]
@justin0309 Pls open a defect against the build you are using, its not a test issue.

**Comment by justin0309** [comment]
@prkatti1
openbmc code base: 2.14.0
auto-test : v4-stable
The error message is still the same
![image](https://github.com/openbmc/openbmc-test-automation/assets/109942943/157a6b0c-78b4-4380-b50f-afb6c2cf50a5)

**Comment by gkeishin** [comment]
@prkatti1 @swe12345 @Anvesh-Kumar-Rayankula  Please check and respond

**Comment by prkatti1** [comment]
Its not a test code issue, Pls open a defect against the build on which you have noticed this issue.

**Comment by justin0309** [comment]
@prkatti1 
what should I do?
I don’t know where he should be classified? bmcweb?

**Comment by prkatti1** [comment]
yes, it is allowing to create user but not allowing to delete user

**Comment by justin0309** [comment]
@prkatti1 
I ran the ```Verify SSH Login Access With Admin User``` test item again. The user is being used and cannot be deleted. Does this make it possible to suspect that the robot test case process needs to be corrected or that there may be other problems?
```Jun 06 08:36:47  phosphor-user-manager[521]: User 'new_admin' deleted successfully
Jun 06 08:36:48  phosphor-user-manager[521]: User 'new_admin' created successfully
Jun 06 08:36:56  phosphor-user-manager[7432]: userdel: user new_admin is currently used by process 8687
Jun 06 08:36:56  phosphor-user-manager[521]: Command /usr/sbin/userdel execution failed, return code 8
Jun 06 08:36:56  phosphor-user-manager[521]: The operation failed internally.
Jun 06 08:36:56  phosphor-user-manager[521]: Delete User 'new_admin' failed
Jun 06 08:36:56  phosphor-user-manager[521]: The operation failed internally.```
It is occupied by this process, but I don’t know who is using it.
```8697 new_admi  3292 S    -sh
```

**Comment by prkatti1** [comment]
Sure, you can push the change.

**Comment by Nandakumar-AMI** [comment]
Hi @gkeishin,

This issue could not be reproduced now. Kindly find the log below,

<img width="1365" height="390" alt="Image" src="https://github.com/user-attachments/assets/3b5bbad6-b5dd-4959-85e3-a2b42a7ac14f" />


The code works and passes without SSHLibrary.Close All Connections. However, it is still recommended to use SSHLibrary.Close All Connections to prevent resource leaks and ensure proper cleanup of SSH sessions by the script.

Additionally, we have the following two lines in the test:
Redfish.Login
Redfish.Delete /redfish/v1/AccountService/Accounts/new_admin

We can remove Redfish.Login as well, since the Test Setup already handles establishing the active root session, and the Test Teardown takes care of logging out. Therefore, an additional Redfish.Login is not required.

Hence, we can close this ticket.

**Comment by fangkaiyu2023** [comment]
Thanks for looking into this. I'd like to highlight one prerequisite that
is essential for reproducing the issue:

Before the deletion operation (e.g., Redfish.Delete
/redfish/v1/AccountService/Accounts/new_admin), the account in question
must have already logged in via SSH.

In other words, the test flow should include an SSH login with that account
before the session is deleted. Without this step, the failure will not
occur, which likely explains why you weren't able to reproduce it.

Could you please try again with the following steps?

   1.

   Establish an SSH session using the target account.
   2.

   Perform the Redfish deletion (or session cleanup).
   3.

   Check whether the error appears.

For reference, my reproduction environment is OpenBMC 2.18.0 release.

Please let me know if you need any additional details.

Best regards

George Keishing ***@***.***> 于2026年7月23日周四 11:57写道：
